frontend/src/lib/lemon-ui/Link/Link.tsx

```diff
@@ -74,6 +78,15 @@ export const Link: React.FC<LinkProps & React.RefAttributes<HTMLElement>> = Reac
             onClickRaw?.(event)
         }
 
+        const MaybeTooltip = ({ children }: { children: JSX.Element }): JSX.Element =>
```

Comment:
It's actually not needed to do this `MaybeTooltip` step here, because `Tooltip` already does it for you, it checks if `title` is empty

Reply:
Thanks! Don't know why I thought I needed that here.